add security measures info (#348)
cncf/tag-contributor-strategy: website/content/contributors/getting-started.md

Many online resources are available to help you learn about Git, pull requests, and issues. Look for online tutorials, documentation, and forums where you can ask questions and get help from other contributors.

#### Understanding ssh keys and Two Factor Authentication

> Contributing to open-source projects requires more than just technical know-how. Knowing about security measures such as SSH keys and two-factor authentication (2FA) is also essential. SSH keys provide a secure way to access remote servers and repositories without entering a password every time, ensuring that user authentication is secure. 2FA adds an extra layer of security by requiring a user to provide two forms of identification before accessing an account or system. For example, this might include a password and a one-time code generated by a mobile app. By requiring contributors to use SSH keys and 2FA, open source projects can help prevent unauthorized access to sensitive data, ensuring that contributions are made securely. For new contributors, it's crucial to familiarize themselves with these security measures before diving in. Resources such as the GitHub documentation on [SSH keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) and [2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) can provide valuable guidance on how to get started.

Reviewer: "For new contributors, it's crucial to familiarize themselves with these security measures before diving in" 

Maybe include why it's crucial? We could mention that GitHub now requires 2FA on any account that submits a pull request (https://github.blog/2023-03-09-raising-the-bar-for-software-security-github-2fa-begins-march-13/). By figuring it out up-front you won't run into trouble submitting a pull request later.

As for SSH keys, that is just one way to authenticate to a repository. I wouldn't say that SSH is more secure than using a GitHub token, for both it depends on how well you secure your private key or password/token. I personally recommend SSH for developers and people comfortable with SSH and the command line because it's just easier _once you have gotten it set up which can be hard for those new to SSH_. People who contribute through UIs like the website, or GitHub desktop, often use other authentication and that's okay too. I care more that they are securely storing their token than anything else (i.e. not embedding it in a plain text file in their git repo).

How about we link them to a couple alternative ways to securely work with GitHub: SSH or Token through the command-line, using the website directly, or GitHub Desktop. Then encourage people to go with what they are comfortable with.

> Contributing to open-source projects requires more than just technical know-how. Knowing about security measures such as SSH keys and two-factor authentication (2FA) is also essential. SSH keys provide a secure way to access remote servers and repositories without entering a password every time, ensuring that user authentication is secure. 2FA adds an extra layer of security by requiring a user to provide two forms of identification before accessing an account or system. For example, this might include a password and a one-time code generated by a mobile app. By requiring contributors to use SSH keys and 2FA, open source projects can help prevent unauthorized access to sensitive data, ensuring that contributions are made securely. For new contributors, it's crucial to familiarize themselves with these security measures before diving in. Resources such as the GitHub documentation on [SSH keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) and [2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) can provide valuable guidance on how to get started.

Author: Good points, thank you @carolynvs
Updated, trying to cover all your feedback

> Contributing to open-source projects requires more than just technical know-how. It's also essential to be aware of security measures such as SSH keys and two-factor authentication (2FA). SSH keys provide a secure way to access remote servers and repositories without the need to enter a password every time, ensuring that user authentication is secure. 2FA adds an extra layer of security by requiring a user to provide two forms of identification before accessing an account or system. This might include a password and a one-time code generated by a mobile app, for example. By requiring contributors to use SSH keys and 2FA, open source projects can help prevent unauthorized access to sensitive data, ensuring that contributions are made securely. For new contributors, it's crucial to familiarize themselves with these security measures before diving in. Resources such as the Github documentation on [SSH keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) and [2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) can provide valuable guidance on how to get started.

Reviewer 2: A quick copy-edit pass:
```suggestion
Contributing to open-source projects requires more than just technical know-how. Knowing about security measures such as SSH keys and two-factor authentication (2FA) is also essential. SSH keys provide a secure way to access remote servers and repositories without entering a password every time, ensuring that user authentication is secure. 2FA adds an extra layer of security by requiring a user to provide two forms of identification before accessing an account or system. For example, this might include a password and a one-time code generated by a mobile app. By requiring contributors to use SSH keys and 2FA, open source projects can help prevent unauthorized access to sensitive data, ensuring that contributions are made securely. For new contributors, it's crucial to familiarize themselves with these security measures before diving in. Resources such as the GitHub documentation on [SSH keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) and [2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) can provide valuable guidance on how to get started.
```

Author: Thank you for the updates
